annotation_full.wdl

```diff
@@ -6,12 +6,13 @@ import "./functional-annotation.wdl" as fa
 workflow annotation {
 input {
   String  proj
-  String  input_file
+  File  input_file
```

Comment:
@scanon  are you okay with this? I know you wanted to test having input files but `File` instead of `String`.

Reply:
What is the reasoning for String vs File again?

Reply:
Best practice is to use `File` but Shane wanted to confirm that doing this didn't impact anything with nmdc automation since previously it was always using `String`.

Reply:
Right now it still needs to be a string so we can pass a URL.  When we move to the new WDL format I think we can use File.

Reply:
reverted to string